api/serializers/personal_info.py

```diff
@@ -65,3 +69,18 @@ class Meta:
             "registration_telegram_bot_chat_id",
             "telegram_username",
         )
+
+
+class DashboardStandalonePersonalInfoSerializer(serializers.ModelSerializer[PersonalInfo]):
```

Comment:
Why "Standalone"? Maybe let's rename it to something like
`MinifiedDashboardPersonalInfoSerializer` as we have in other serializers?